parliament/migrations/0002_parliamentmember.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.3 on 2017-04-25 17:58
+from __future__ import unicode_literals
```

Comment:
This is auto generated by django, ill create an issue and someone can look into it